gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
not sure why this changed but it's ok

Reply:
Hm, I recall I did upgrade Gradle whilst working on this, but don't recall manually changing this at all

Reply:
It was probably automatic, but is should be fine to upgrade gradle